tests/review-child-handoff.test.sh

```
@@ -48,6 +48,279 @@ simplify_context_out="$(uberdev_agent_context_create "$TMP/simplify-run" "$simpl
+    else
+      command "$REAL_PORTABLE_PYTHON" "$@"
+    fi
+  }
+
+  portable_manifest="$ROOT/tests/_fixtures/.portable-child-run-tree-$$.json"
+  portable_manifest_swap="$portable_manifest.swap"
+  cp "$ROOT/plugins/uberdev/policy/solve-run-tree-v1.json" "$portable_manifest"
+  trap 'rm -f "$portable_manifest" "$portable_manifest_swap"' EXIT
+  portable_manifest_real="$(cd "$(dirname "$portable_manifest")" && pwd -P)/$(basename "$portable_manifest")"
+  [ "$(UBERDEV_CHILD_TEST_MODE=1 UBERDEV_CHILD_MANIFEST_PATH="$portable_manifest" _uberdev_child_manifest_path)" = "$portable_manifest_real" ]
+  ! UBERDEV_CHILD_TEST_MODE=1 UBERDEV_CHILD_MANIFEST_PATH="$portable_manifest" \
```

Shellcheck (0.11.0)

[info] This ! is not on a condition and skips errexit. Use `&& exit 1` instead, or make sure $? is checked.

(SC2251)